services/terrain/src/terrain/clients/iplant_groups.clj

```diff
@@ -10,6 +10,7 @@
   {:username (:id response)
    :firstname (:first_name response)
    :lastname (:last_name response)
+   :name (:name response)
```

Comment:
Always in favor of being a little bit less wrong as far as the entries in https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/ :D